avocado/utils/network/interfaces.py

```diff
@@ -62,7 +62,7 @@ def config_filename(self):
         return "{}/ifcfg-{}".format(path, self.name)
 
     def _get_interface_details(self, version=None):
-        cmd = "ip -j link show {}".format(self.name)
+        cmd = "ip -d -j link show {}".format(self.name)
```

Comment:
`-d` tag gives us more detail like the vlan number tag.

Reply:
Note for the next PR: You can explain why you are adding this with a different commit message (or using the same commit if that makes sense)